src/app/features/courses/pipes/order-by.pipe.mock.ts

```diff
@@ -0,0 +1,12 @@
+import {Pipe, PipeTransform} from '@angular/core';
+import { CoursesItemModel } from '../models/courses-item.model';
+
+@Pipe({
+  name: 'orderBy'
+})
+
+export class OrderByMockPipe implements PipeTransform {
+    transform(value: CoursesItemModel[]): CoursesItemModel[] {
+        return [];
+    }
+}
```

Comment:
Where are you use it?

Reply:
The pipe is used in the courses page component template.
the mock is used in the following spec: 
src\app\features\courses\containers\courses-page\courses-page.component.spec.ts